packages/vaadin-charts/test/chart-properties.test.js

```diff
@@ -261,6 +262,15 @@ describe('vaadin-chart properties', () => {
       chart.addEventListener('xaxes-extremes-set', extremesListener);
       chart.setAttribute('category-min', newCategoryMin);
     });
+
+    it('should warn when setting a not valid minimum value', () => {
```

Comment:
It was an attempt to improve the chart's code coverage, but it didn't significantly change the numbers. After all, I decided to keep it here anyway.